client/package.json

```diff
@@ -175,7 +178,29 @@
       "no-underscore-dangle": 0,
       "spaced-comment": "off",
       "no-useless-constructor": "off",
+      "no-use-before-define": "off",
+      "vue/require-valid-default-prop": "off",
+      "vue/require-default-prop": "off",
+      "vue/no-unused-components": "off",
+      "vue/valid-v-slot": "off",
+      "vuejs-accessibility/alt-text": "off",
+      "vue/no-v-text-v-html-on-component": "off",
+      "@typescript-eslint/no-unused-vars": "off",
+      "vue/no-template-target-blank": "off",
+      "vue/no-lone-template": "off",
+      "default-param-last": "off",
```

Comment:
I'm a fan of `default-param-last`. Could we just disable it inline if it's needed for API stability reasons in a few places?

Reply:
When I updated the base eslint rules there were a bunch of new issues being thrown.  This one was small enough I can resolve it fairly quickly.  Most instances were older functions with default params that required more params added for newer features.  There were only a few instances so I removed this and added in sane defaults.